opencti-platform/opencti-front/src/components/list_lines/ListLinesContent.jsx

```diff
@@ -268,7 +268,7 @@ class ListLinesContent extends Component {
                           registerChild(ref);
                         }}
                         autoHeight={true}
-                        height={height}
+                        height={height || 0}
```

Comment:
height is undefined when there is no data to list, showing a warning in the console.